ramls/traits/sortable.raml

```diff
@@ -4,5 +4,4 @@ queryParameters:
     type: string
     description: Option by which results are sorted. Defaults to relevance if query or name if no query. Possible values are name, relevance.
     example : name
-    default: relevance
```

Comment:
curious why this is removed from raml?

Reply:
there was a very minor problem:
ruby module returns correct response for this path
/eholdings/providers
but returns an error for this path
/eholdings/providers?sort=relevance

if default is set to relevance then we can't distinguish between those two requests.

now that I think about it, maybe it's better to just ignore sort parameter in case of
/eholdings/providers?sort=relevance
and treat it as /eholdings/providers

what do you think?

Reply:
Took a look at specific error we are seeing in the case of `/eholdings/providers?sort=relevance`
It is returning the error from rm api
`Parameter OrderBy should be by ProviderName when parameter Search is empty.` 
and from RM API documentation -- `Search Term - When searching for vendors without a search parameter, sort options will be by vendorname. When the search parameter is not null, the default sort is by relevance.`
So default sort will depend on the presence of a q parameter